tests/bearlib/aspects/ClassTest.py

```diff
@@ -1,5 +1,3 @@
-from coalib.bearlib.aspects import aspectclass
```

Comment:
Please don't tamper with aspectclasses, they are currently in development state and coala analysis should be done on them once they are up and running. These imports might be added keeping in mind to add more functionality later.